assets/js/analytics-events.tsx

```diff
@@ -192,23 +193,38 @@ function SidebarNavItem({ name }: { name: string }) {
   );
 }
 
-function ErrorPage({ error, url }: { error: Error; url: string }) {
+function AlertComponent({
+  heading,
+  content,
+}: {
+  heading?: string;
+  content?: VNode | string;
+}) {
   return (
     <div className="usa-alert usa-alert--error">
       <div className="usa-alert__body">
-        <h5 className="usa-alert__heading">Error loading event definitions</h5>
-        <div className="usa-alert__text">
-          <p>
-            There was an error loading event definitions from{" "}
-            <a href={url}>{url}</a>:
-          </p>
-          <p>{error.message}</p>
-        </div>
+        <h5 className="usa-alert__heading">{heading}</h5>
+        <div className="usa-alert__text">{content}</div>
       </div>
     </div>
   );
 }
 
+function ErrorPage({ error, url }: { error: Error; url: string }) {
+  return AlertComponent({
```

Comment:
Should we be calling this as a component?

```suggestion
  return <AlertComponent
```